index.html

```diff
@@ -183,7 +184,45 @@ <h4>Skills</h4>
                 </ul>
             </div>
         </div> 
+    </div>
     </section>
-</section>
+    <footer class="form">
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h3>
+        <div class="form-section">
+            <form action="https://formspree.io/f/myyvjjng" method="post" id="form">
+                <div class="form-input">
+                    <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    <input type="email"  id="email" name="user_email"  placeholder="Email address" required>
+                    <textarea type="text"  id="message" name="user_message"  maxlength="500" required></textarea>
+                </div>
+                <div class="back-img4"></div>
+                <div class="get-in-touch-btn">
+                    <button class="form-btn" type="submit">Get in touch </button>
+                </div>
+            
+            </form>
```

Comment:
- Kindly replace the `footer` tag with a `section` tag. The `footer` tag defines a footer for a document or section. A footer element typically contains authorship information, copyright information, contact information, sitemap, back to top links. The footer should not contain a form. :nerd_face: :nerd_face: 

- Kindly make sure that your project sticks as much as possible to the Figma template provided. Please add the placeholder to the `textarea` tag as the Figma template shows. :pray: :pray: 

Your design:
![Screenshot from 2022-10-05 07-10-33](https://user-images.githubusercontent.com/98363075/194058370-c0f79c44-0f67-4694-b7c5-e965bf8daff4.png)

Figma design:
![Screenshot from 2022-10-05 07-12-42](https://user-images.githubusercontent.com/98363075/194058418-c705c404-35d8-4cc8-b4ee-4058ced1c548.png)